src/fontra/views/applicationsettings/applicationsettings.js

```diff
@@ -0,0 +1,116 @@
+import * as html from "../core/html-utils.js";
+import { ServerInfoPanel } from "./panel-server-info.js";
+import { ShortCutsPanel } from "./panel-shortcuts.js";
+import { ensureLanguageHasLoaded, translate } from "/core/localization.js";
+import { message } from "/web-components/modal-dialog.js";
+
+export class ApplicationSettingsController {
+  static async fromWebSocket() {
```

Comment:
This method name makes no sense, as there is no websocket involved. 

Perhaps the html script should call `.start()`, and the `start()` method should do `await ensureLanguageHasLoaded`.

Reply:
Thanks. done.